README.md

```diff
@@ -507,6 +513,14 @@ An Array of the `Worker` instances used by this pool.
 
 The current number of tasks waiting to be assigned to a Worker thread.
 
+### Property: `needsDrain` (readonly)
```

Comment:
I think we don't need it anymore

Reply:
but, wouldn't that be the case if the feature just subscribes to pool.on('drain')? am I missing something?

Reply:
Yes you are missing something. The drain event only means that needaDrain goes from true to false, you still need the property for the other way around.

Reply:
Oh, right. Maybe the terminology isn't clear enough.